### PR TITLE
chore: fix warnings on playground pages

### LINKED ISSUE #325: Improve Docs: Cleanup warnings on playground
### Description

Many components are throwing errors and warnings on playground pages.

### Versions
**fundamental-react:**
**fiori-fundamentals:**

---

_**NOTE:** Where applicable, please include uncropped screen captures._

_**DISCLAIMER:**
After triaging an issue, the fundamental-react team will see if it can be reproduced or confirmed. If more information is needed, the fundamental-react team will contact the author.  Any issues awaiting responses from the author for more than 7 days will be closed.  The author can re-open the issue at a later time if they can present the requested information._

### LINKED ISSUE #360: Dropdown Component: isContextual prop missing
### Description

The prop `isContextual` is used several times in the playground for the Dropdown component. The Dropdown component does not have an `isContextual` prop. 

https://github.com/SAP/fundamental-react/blob/master/src/Dropdown/Dropdown.js#L5

https://github.com/SAP/fundamental-react/blob/master/src/_playground/documentation/Playground/Playground.js#L383

What was this prop meant to do? 
### Versions
**fundamental-react:** 0.3.1
**fiori-fundamentals:**

---

_**NOTE:** Where applicable, please include uncropped screen captures._

_**DISCLAIMER:**
After triaging an issue, the fundamental-react team will see if it can be reproduced or confirmed. If more information is needed, the fundamental-react team will contact the author.  Any issues awaiting responses from the author for more than 7 days will be closed.  The author can re-open the issue at a later time if they can present the requested information._

### PR DESCRIPTION
**Pull request by jbadan**
### Description

Many of the playground pages had console warnings due to the Description component and several other small issues. 

The only remaining warnings will be solved by #359 

fixes #325 
fixes #360

### CODE REVIEW

File: src/Button/Button.Component.js [hunk 1]
```diff
@@ -110,6 +110,22 @@ const clickBtnHandler = btn => {
         alert(`You clicked the ${btn} Button`);
     };
 
+    const buttonOptionsDescription = `
+There are three emphasis styles used to indicate the importance of the button on the page. \n\n
+* **Emphasized Button**: There should only be one highlighted button on the page. This is
+the primary call to action.\n\n
+* **Regular Button**: The default button style and the most common button. There may be
```
Comment (jbadan) [inline src/Button/Button.Component.js]: @mobot11 Is `shell` now the same as the regular/default option?

File: src/Button/Button.Component.js [hunk 2]
```diff
@@ -151,15 +159,7 @@ const clickBtnHandler = btn => {
...
-            <Description>
-                <strong> - Action Button</strong>: The default button <br />
-                <strong> - Standard Button</strong>: Neutral or informative color <br />
-                <strong> - Positive Button</strong>: Used for positive actions such as approved, ok, yes. <br />
-                <strong> - Medium Button</strong>: Used for warnings or alert <br />
-                <strong> - Negative Button</strong>
-                :Used for negative actions such as decline, cancel, no.
-                <br />
-            </Description>
+            <Description children={buttonTypesDescription} />
```
Comment (greg-a-smith) [inline src/Button/Button.Component.js]: Although I know this works, is it clearer/cleaner/more standard to just place the variable inside the JSX tags?
```jsx
<Description>{buttonTypesDescription}</Description>
```
Same comment on a few other instances below.

File: src/_playground/App.scss [hunk 3]
```diff
@@ -113,7 +113,7 @@ $fd-fonts-path: '../../node_modules/fiori-fundamentals/dist/fonts/';
                 transition: background-color .25s;
             }
 
-            .frDocs-markdown {
+            .frDocs-markdown, .frDocs-Content__description {
```
Comment (greg-a-smith) [inline src/_playground/App.scss]: Do we need another class or just add the `frDocs-markdown` class to the `ReactMarkdown` component in `Description`?

File: src/Button/Button.Component.js [hunk 4]
```diff
@@ -238,7 +238,7 @@ const clickBtnHandler = btn => {
 
             <h2>Button States</h2>
             <Description>
-                There are three states: <code>normal</code> (default), <code>selected</code>, and <code>disabled</code>.
+                There are three states: `normal` (default), `selected`, and `disabled`.
```
Comment (jbadan) [inline src/Button/Button.Component.js]: @mobot11 is this still accurate? Or is it just default, `selected` and `disabled`?

File: src/Button/Button.Component.js (hunk 1 above)
Comment (greg-a-smith) [reply src/Button/Button.Component.js]: I think `shell` is leftover from some previous iterations.  Adding `option='shell'` _does_ do some alternate styling, but the button is very faint and does not look usable.  Furthermore, the SAP Fiori Fundamentals site shows only three options for Button:  **Emphasized**, **Regular** and **Light** and the component is marked "Stable" (see screenshot below).  I think we should remove `shell` from the list of available options for a button.

screen shot 2019-02-06 at 10 16 29 am

**Comment by greg-a-smith** [comment]
I will approve again ⛵️ , but I'm now part author.  Can someone else review this?

File: src/Button/Button.Component.js (hunk 1 above)
Comment (mobot11) [reply src/Button/Button.Component.js]: I think we're good to remove `shell` as an option.

File: src/Button/Button.Component.js (hunk 4 above)
Comment (mobot11) [reply src/Button/Button.Component.js]: Looks like a button can be both selected and disabled
```javascript
    const buttonClasses = classnames(
        {
            'fd-button': !option,
            [`fd-button--${option}`]: !!option,
            [`fd-button--${type}`]: !!type,
            'fd-dropdown__control': dropdown,
            'fd-button--compact': compact,
            [`sap-icon--${glyph}`]: !!glyph,
            'fd-global-nav__btn': navbar,
            'is-selected': selected,
            'is-disabled': disabled
        },
        className
    );
```

But adding normal as a prop would have no effect on how the component actually renders. However fundamentals does describe the default state as normal in their documentation so I'd leave this as is to stay synced up with fundamentals https://sap.github.io/fundamental/components/button.html#standard-button-state

**Comment by greg-a-smith** [comment]
Per @jbadan's request, I am merging this on her behalf.

**Comment by jbadan** [comment]
🚢

**Pull request merged**